### PR TITLE
docs: add features and output section in readme

### PR DESCRIPTION
**Pull request by aayushmau5**
**Description**

- Added Features and Output section in README

### CODE REVIEW

**Comment by aayushmau5** [comment]
@derberg Are we mission something in the README? Let me know, I'll add them as well.

File: README.md [hunk 1]
```diff
@@ -90,6 +95,33 @@ const output = diff(firstDocument, secondDocument, {
 });
 ```
 
+## Output
```
Comment (vinitshahdeo) [inline README.md:98]: @aayushmau5 Not sure, maybe we can present the different possible values in a tabular form?

Comment (magicmatatjahu) [reply README.md:98]: @vinitshahdeo What do you mean by `tabular form` 🤔 ?

File: README.md [hunk 2]
```diff
@@ -90,6 +95,33 @@ const output = diff(firstDocument, secondDocument, {
...
+
+This library supports JSON(`json`), YAML(`yaml` | `yml`) & Markdown(`markdown` | `md`) output.
```
Comment (magicmatatjahu) [inline README.md:100]: I think that list will be better:

```suggestion
This library supports outputs:

- JSON: `json`
- YAML: `yaml` or `yml`
- Markdown: `markdown` or `md`
```

**Comment by derberg** [comment]
lgtm, just not sure what @vinitshahdeo meant about tabular view

**Pull request merged**